src/Servers/Kestrel/test/Interop.FunctionalTests/Http3/Http3RequestTests.cs

```diff
@@ -514,6 +514,69 @@ public async Task GET_MultipleRequestsInSequence_ReusedState()
             }
         }
 
+        [ConditionalFact]
+        [MsQuicSupported]
+        public async Task StreamResponseContent_DelayAndTrailers_ClientSuccess()
```

Comment:
Un-related unit test. IMO no harm in adding it. These tests are all quarantined.